### PR TITLE
Remove unused

### PR DESCRIPTION
**Pull request by crewtool**
# Description
removing old files/archive

# Checklist
   - [ ] Is this PR focused on a single feature without the possibility of being divided into smaller PRs?
   - [ ] Are all variables set with logical defaults that are backward compatible?
   - [ ] I attached link to neptune if it was necessary
   - [ ] Have you successfully executed the linter and tests?
   - [ ] Are all functions concise and don't require splitting?
   - [ ] Is there documentation for arguments and complex logic?
   - [ ] Are there no temporary solutions in the code? Should a task be added to Trello to refine the code?
   - [ ] Are all functions placed in appropriate files?
   - [ ] Is the PR name meaningful and descriptive?
   - [ ] Is PR description provided, or unnecessary?
  
Other questions:
   - [ ] If I made changes to the requirements or anything related to the Singularity image I did generate new image and upload a new image to the clusters
   - [ ] If this change is significant enough I notified all individuals working with this repository (@channel in slack channel)
   - [ ] If there is a need for a second reviewer I requested additional review 

# Reviewer's checklist:
   - [ ] Does the Neptune link work and does it actually compare the situation before and after the PR?
   - [ ] Do I understand the code?
   - [ ] Do I think another reviewer is not needed or I commented that this PR need another reviewer?

### CODE REVIEW

File: lizrd/support/ash.py [hunk 1]
```diff
@@ -4,10 +4,7 @@
...
 
 
-def assert_shape(pattern, tensor, **kwargs):
-    if DISABLE_CHECKS:
-        return
-    einops.rearrange(tensor, f"{pattern} -> {pattern}", **kwargs)
+
 
 
 class Check(nn.Module):
```
Comment (Krystian-Krol) [inline lizrd/support/ash.py:10]: is this file needed anymore if you remove both `assert_shape` and `check`?

File: research/conditional/moe_layers/cont_moe_designs/send_result_only_to_top1_token.py [hunk 2]
```diff
@@ -1,11 +1,27 @@
+import torch
 from lizrd.core import misc
-from lizrd.support import ash
-from research.conditional.archive.rogue_code import set_highest_index_one
 from research.conditional.moe_layers.continuous_moe import ContinuousMoeBaseClass
 from research.conditional.utils.misc_tools import stable_softmax_temperature
 
 
-@ash.check("... dinp -> ... dout")
+def set_highest_index_one(tensor: torch.Tensor) -> torch.Tensor:
```
Comment (Krystian-Krol) [inline research/conditional/moe_layers/cont_moe_designs/send_result_only_to_top1_token.py:7]: is it used anywhere or spill from other PR?

**Review by Krystian-Krol — COMMENTED** [review]
seems fine,  but I don't have expertise on this piece of code (it's from before I joined)

File: research/conditional/moe_layers/cont_moe_designs/send_result_only_to_top1_token.py (hunk 2 above)
Comment (crewtool) [reply research/conditional/moe_layers/cont_moe_designs/send_result_only_to_top1_token.py:7]: Apparently class below was using this function from archive

Comment (crewtool) [reply research/conditional/moe_layers/cont_moe_designs/send_result_only_to_top1_token.py:7]: So I moved it here and removed archive file

File: research/conditional/moe_layers/ffs.py [hunk 3]
```diff
@@ -74,7 +79,6 @@ def forward(self, x):
         # that we need for
         with Timer("rewrittenFF", disable_inner=False):
             # BATCH, embedding
-            ash.assert_shape("... B d", x, d=self.dm)
```
Comment (crewtool) [inline research/conditional/moe_layers/ffs.py:77]: TODO(me)

**Review by sebastianjaszczur — APPROVED** [review]
RIP ash check, you were too beautiful for this cruel world [*]
LGTM
